docs/howto/account-billing/change-credit-card.md

```diff
@@ -10,16 +10,18 @@ account to switch credit cards.
 
 ## Change credit card
 
-Navigate to the Dashboard list in the left-hand menu and click on _Invoices_.
+In the {{gui}}, make sure the vertical pane on the left is visible.
+Click on _Settings_, and from the drop-down menu that appears select _Manage Invoice Settings_.
 
-![White calculator and the word invoice on orange background.](assets/dashboardinvoice.png)
-
-Now you will see your invoice specifications. Click on the _circle
-with three dots_ beside the invoice you want to pay.
+Your invoice specifications are in the _Settings_ tab. To pay an invoice, click on the orange :material-dots-horizontal-circle: icon beside it.
```

Comment:
Micro-nit: two sentences should use two lines.
```suggestion
Your invoice specifications are in the _Settings_ tab.
To pay an invoice, click on the orange :material-dots-horizontal-circle: icon beside it.
```

Reply:
Ah, yes.